docs/layouts/shortcodes/generated/kubernetes_operator_config_configuration.html

```diff
@@ -112,7 +112,7 @@
         </tr>
         <tr>
             <td><h5>kubernetes.operator.reconcile.parallelism</h5></td>
-            <td style="word-wrap: break-word;">5</td>
+            <td style="word-wrap: break-word;">10</td>
```

Comment:
why do we need to increase the parallelism?

Reply:
We use the default from the josdk we don’t set this value explicitly anywhere

Reply:
makes sense, didn't notice it